upi/openstack/inventory.yaml

```diff
@@ -11,6 +11,8 @@ all:
       os_flavor_worker: 'm1.large'
       os_image_rhcos: 'rhcos'
       os_external_network: 'external'
+      os_api_fip: ''
+      os_ingress_fip: ''
```

Comment:
I am not sure what the user should insert here

Reply:
This should be the floating IP addresses that were created in step https://github.com/openshift/installer/blob/master/docs/user/openstack/install_upi.md#api-and-ingress-floating-ip-addresses.

Reply:
Do you think it's worth adding a small comment stating that?